src/fallback/index.ts

```diff
@@ -7,6 +7,10 @@ export const us_listen_socket_close = (listenSocket: any) => {
   return listenSocket.close();
 };
 
+export const us_socket_local_port = (listenSocket: any) => {
+  return listenSocket.address().port;
```

Comment:
What if `address()` is null?

Reply:
Handled here: https://github.com/trufflesuite/uws-js-unofficial/pull/28/commits/b66be707216a05b2200006f202bc2d5947872503